FRDeviceBinding/FRDeviceBindingTests/DeviceAuthenticatorTests.swift

```diff
@@ -455,4 +455,166 @@ class DeviceAuthenticatorTests: FRBaseTestCase {
             //all good, do nothing
         }
     }
+    
+    
+    func test_14_ValidateCustomClaims_valid() {
```

Comment:
Add a test case that overrides the validateCustomClaims method, the validateCustomClaims method just return true, and when doing the binding or signing, the developer wants to override one of the existing claim.

Reply:
@witrisna Added two custom classes, one always return true for validateCustomClaims and one always return false.
Added unit tests for those